docs/eventing/samples/kafka/channel/README.md

```diff
@@ -1,209 +1,246 @@
----
-title: "Apache Kafka Channel Example"
-linkTitle: "Channel Example"
-weight: 20
-type: "docs"
----
-
 # Apache Kafka Channel Example
 
-You can install and configure the Apache Kafka CRD (`KafkaChannel`) as the
-default channel configuration in Knative Eventing.
+You can install and configure the Apache Kafka Channel as the
+default Channel configuration for Knative Eventing.
 
 ## Prerequisites
 
-- Ensure that you meet the
-  [prerequisites listed in the Apache Kafka overview](../).
 - A Kubernetes cluster with
   [Knative Kafka Channel installed](../../../../admin/install/).
```

Comment:
can the link be replaced with https://knative.dev/docs/admin/install/eventing/install-eventing-with-yaml/#optional-install-a-default-channel-messaging-layer instead so that it directly lands on the page where a channel layer is installed?

and also one more prerequisite of installing the broker layer since its being used in this sample? (mt channel based) https://knative.dev/docs/admin/install/eventing/install-eventing-with-yaml/#optional-install-a-broker-layer

Reply:
I don't want to get too deep into adding specific links, because I have a feeling we'll move these samples to a better place and also maybe do some work on splitting out the optional stuff from the eventing install later. For now I've updated the prereqs to say you need to install eventing and the optional broker and kafka channel components. I think it's better to keep it high level for now.